app/views/mobile-client-builds-row.html

```diff
@@ -50,30 +53,30 @@
       <div class="expanded-section no-margin">
         <div class="component-label section-label">BUILD CONFIG</div>
         <div class="row">
-          <div class="col-md-2">
+          <div class="col-sm-6 col-md-2">
             <b>Repo Url</b>
           </div>
-          <div class="col-md-4">
+          <div class="col-sm-6 col-md-4">
             {{row.apiObject.spec.source.git.uri}}
           </div>
-          <div class="col-md-2">
+          <div class="col-sm-6 col-md-2">
             <b>Branch</b>
           </div>
-          <div class="col-md-4">
+          <div class="col-sm-6 col-md-4">
             {{row.apiObject.spec.source.git.ref}}
           </div>
         </div>
         <div class="row">
-          <div class="col-md-2">
-            <b>Jenkins Job Name</b>
+          <div class="col-sm-6 col-md-2">
+            <b>Build Platform</b>
           </div>
-          <div class="col-md-4">
-            {{row.apiObject.metadata.name}}
+          <div class="col-sm-6 col-md-4">
+            {{row.apiObject.spec.buildPlatform}}
```

Comment:
I think buildPlatform in spec is not supported by the API. We may need to add this as a label instead.